release-notes/7.0/preview/api-diff/preview5/Microsoft.AspNetCore.App/7.0-preview5_Microsoft.JSInterop.Infrastructure.md

```diff
@@ -0,0 +1,8 @@
+# Microsoft.JSInterop.Infrastructure
+
+``` diff
+ namespace Microsoft.JSInterop.Infrastructure {
+     public static class DotNetDispatcher
```

Comment:
This one is odd, did the tool break?
I assume it's trying to show the attribute changes from https://github.com/dotnet/aspnetcore/commit/442a380854c25b85252a743c3ba2e21f08fa6879#diff-ee83750b0c4b366580d485516093fb99c3c9b1fce10eef52846211bcb9e9021e

Reply:
> I assume it's trying to show the attribute changes

Yes, that's what happened. It's missing the attribute change, and it also did not add the empty `{}` to indicate no members were modified.

Reply:
```suggestion
     public static class DotNetDispatcher
     {
+        [System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute("Trimming", "IL2026", Justification = "We expect application code is configured to ensure return types of JSInvokable methods are retained.")]
         public static string? Invoke(Microsoft.JSInterop.JSRuntime jsRuntime, in Microsoft.JSInterop.Infrastructure.DotNetInvocationInfo invocationInfo, string argsJson);
+        [System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute("Trimming", "IL2026", Justification = "We expect application code is configured to ensure return types of JSInvokable methods are retained.")]
         public static void BeginInvokeDotNet(Microsoft.JSInterop.JSRuntime jsRuntime, Microsoft.JSInterop.Infrastructure.DotNetInvocationInfo invocationInfo, string argsJson)
     }
```